pyre/__init__.py

```diff
@@ -1,7 +1,6 @@
 __all__ = ['pyre', 'zbeacon', 'zhelper']
-
-from pkg_resources import resource_string
-__version__ = resource_string('pyre', 'VERSION')
+__version__ = '0.3.2'
+__version_info__ = tuple(int(v) for v in __version__.split('.'))
```

Comment:
Depends on what we expect the version method to return.
Also can't have versions like `0.3.2-dev` with ints.